podman-compose.yml

```diff
@@ -8,6 +8,6 @@ services:
      - "137:137/udp"
      - "138:138/udp"
     volumes:
-     - /opt/samba/etc/samba:/etc/samba:Z
+     - /opt/samba-config.json:/etc/samba-config-json:Z
```

Comment:
After creating proper samba-config.json I would have to keep this file with user credentials on hard disk unencrypted, which I would like to avoid. Is there any idea how to do this?